### PR TITLE
CU-26aekff Add custom link applicable footnote

### PR DESCRIPTION
**Pull request by ronahk**
A0A1CF57-3BAF-4340-B8F4-C478B9F7712B

### CODE REVIEW

File: docs/Learn/Annotations.md [hunk 1]
```diff
@@ -7,112 +7,94 @@ Komodor annotations should be placed in the deployment resource annotations (ann
...
-|${namespace} | Namespace Name|
-|${cluster} | Cluster Name|
-|${failedPod} | The pod name of a failed pod that triggered this health event* |
-|${container[<name\>].image} | Image name of a container| ${container[web].image}
-|${timestampStart=yyyy-MM-dd'T'HH:mm:ss.SSS} | Start Time in custom format**| ${timestampStart=yyyy-MM-dd}
-|${timestampEnd=yyyy-MM-dd'T'HH:mm:ss.SSS} | End Time in custom format**| ${timestampEnd=yyyy-MM-dd}
-|${yaml[<spec_path\>]} | Full yaml's path specification | ${yaml[metadata.labels.app]}
+| Placeholder                                   | Value                                                           | Example                      |
+| --------------------------------------------- | --------------------------------------------------------------- | ---------------------------- |
+| ${epochStart}\*                               | Start Time in Epoch Time                                        |
```
Comment (gxxglyeyez) [inline docs/Learn/Annotations.md]: add a space before \* 
like so:
${epochStart} \*

**Review by omrisavra — APPROVED** [review]

File: docs/Learn/Annotations.md [hunk 2]
```diff
@@ -135,15 +117,15 @@ spec:
...
 
-| YAML Path                                      | Value         | Explanation                             |
-|------------------------------------------------|---------------|-----------------------------------------|
-|{$yaml[spec.replicas]}                          | 5             | full path usage                         |
-|{$yaml[spec.template.spec.containers[0\].name]} | test          | full path usage using path index        |
-|{$yaml[spec.my_replicas]}                       | undefined     | path doesn't exist                      |
-|{$yaml[spec.template.spec.containers]}          | undefined     | path doesn't resolve to an actual value |
+| YAML Path                                       | Value     | Explanation                             |
+| ----------------------------------------------- | --------- | --------------------------------------- |
+| {$yaml[spec.replicas]}                          | 5         | full path usage                         |
```
Comment (gxxglyeyez) [inline docs/Learn/Annotations.md:122]: the placeholder should be ${ and not {$

**Review by gxxglyeyez — COMMENTED** [review]
minor changes required

File: docs/Learn/Annotations.md (hunk 1 above)
Comment (ronahk) [reply docs/Learn/Annotations.md]: done

**Pull request merged**